lib/utils/plugins.js

```diff
@@ -166,85 +179,44 @@ export function reload () {
   decorated = {};
 }
 
-export function getTermProps (uid, parentProps, props) {
+function getProps (name, props, ...fnArgs) {
+  const decorators = propsDecorators[name];
   let props_;
 
-  termPropsDecorators.forEach((fn) => {
+  decorators.forEach((fn) => {
     let ret_;
 
     if (!props_) props_ = Object.assign({}, props);
 
     try {
-      ret_ = fn(uid, parentProps, props_);
+      ret_ = fn(...fnArgs, props_);
     } catch (err) {
       console.error(err.stack);
-      notify('Plugin error', `${fn._pluginName}: Error occurred in \`getTermProps\`. Check Developer Tools for details.`);
+      notify('Plugin error', `${fn._pluginName}: Error occurred in \`${name}\`. Check Developer Tools for details.`);
       return;
     }
 
     if (!ret_ || 'object' !== typeof ret_) {
-      notify('Plugin error', `${fn._pluginName}: Invalid return value of \`getTermProps\` (object expected).`);
+      notify('Plugin error', `${fn._pluginName}: Invalid return value of \`${name}\` (object expected).`);
       return;
     }
 
-    props = ret_;
```

Comment:
`getTermProps` was setting `props = ret_`, unlike the others which did `props_ = ret_`. Think that's a bug?